TestCollector is flaky, again

```

2016/02/05 10:59:21 Creating pipe id foo

2016/02/05 10:59:21 Timing out pipe foo

2016/02/05 10:59:21 Pipe pipe-5577006791947779410 connection to 127.0.0.1:51173 starting

2016/02/05 10:59:21 Creating pipe id pipe-5577006791947779410

2016/02/05 10:59:21 Closing pipe pipe-5577006791947779410

2016/02/05 10:59:21 Pipe pipe-5577006791947779410 connection to 127.0.0.1:51173 exiting

--- FAIL: TestCollector (0.01s)

	collector_test.go:38: 

		--- want

		+++ have

		@@ -1,28 +1,7 @@

		 (report.Report) {

		  Endpoint: (report.Topology) {

		-  Nodes: (report.Nodes) (len=2) {

		+  Nodes: (report.Nodes) (len=1) {

		    (string) (len=3) "bar": (report.Node) {

		-    ID: (string) "",

		-    Topology: (string) "",

		-    Counters: (report.Counters) {}

		-,

		-    Sets: (report.Sets) {}

		-,

		-    Adjacency: (report.IDList) <nil>,

		-    Edges: (report.EdgeMetadatas) {}

		-,

		-    Controls: (report.NodeControls) {

		-     Timestamp: (time.Time) 0001-01-01 00:00:00 +0000 UTC,

		-     Controls: (report.StringSet) <nil>

		-    },

		-    Latest: (report.LatestMap) {}

		-,

		-    Metrics: (report.Metrics) {

		-    },

		-    Parents: (report.Sets) {}

		-

		-   },

		-   (string) (len=3) "foo": (report.Node) {

		     ID: (string) "",

		     Topology: (string) "",

		     Counters: (report.Counters) {}

		@@ -101,6 +80,6 @@

		  },

		  Window: (time.Duration) 0,

		  Shortcut: (bool) false,

		- ID: (string) (len=19) "4324745483838182873"

		+ ID: (string) (len=16) "2734628a7a6460ef"

		 }

		 

		

2016/02/05 10:59:22 Control connection to 127.0.0.1:56294 starting

2016/02/05 10:59:22 Closing control connection to 127.0.0.1:56294

2016/02/05 10:59:22 Closing control connection to 127.0.0.1:56294

2016/02/05 10:59:22 Control connection to 127.0.0.1:56294 exiting

FAIL

coverage: 45.3% of statements in github.com/weaveworks/scope/app, github.com/weaveworks/scope/common/exec, github.com/weaveworks/scope/common/fs, github.com/weaveworks/scope/common/hostname, github.com/weaveworks/scope/common/marshal, github.com/weaveworks/scope/common/mtime, github.com/weaveworks/scope/common/sanitize, github.com/weaveworks/scope/common/xfer, github.com/weaveworks/scope/probe, github.com/weaveworks/scope/probe/appclient, github.com/weaveworks/scope/probe/controls, github.com/weaveworks/scope/probe/docker, github.com/weaveworks/scope/probe/endpoint, github.com/weaveworks/scope/probe/endpoint/procspy, github.com/weaveworks/scope/probe/host, github.com/weaveworks/scope/probe/kubernetes, github.com/weaveworks/scope/probe/overlay, github.com/weaveworks/scope/probe/process, github.com/weaveworks/scope/render, github.com/weaveworks/scope/render/detailed, github.com/weaveworks/scope/report

FAIL	github.com/weaveworks/scope/app	1.319s
```

--- a/app/collector_test.go
+++ b/app/collector_test.go
@@ -14,7 +14,7 @@ import (
 
 func TestCollector(t *testing.T) {
 	ctx := context.Background()
-	window := time.Millisecond
+	window := 10 * time.Second
 	c := app.NewCollector(window)
 
 	r1 := report.MakeReport()
@@ -33,7 +33,6 @@ func TestCollector(t *testing.T) {
 	}
 
 	c.Add(ctx, r2)
-
 	merged := report.MakeReport()
 	merged = merged.Merge(r1)
 	merged = merged.Merge(r2)

